qalculate/src/extension.cpp

```diff
@@ -123,9 +123,11 @@ void Qalculate::Extension::handleQuery(Core::Query * query) const {
 
     QString result, error, cmd = query->string().trimmed();
     MathStructure mathStructure;
+
     try {
         string expr = d->calculator->unlocalizeExpression(query->string().toStdString(), d->eo.parse_options);
         mathStructure = d->calculator->calculate(expr, d->eo);
+        mathStructure.setPrecision(d->calculator->getPrecision());
```

Comment:
why is this needed?

Reply:
As the main precision setting doesn't seem to work correctly thus using extremely large precision (just try to type in "pi")

Reply:
whats the version you are using? seems like most OS are still with 0.9

Reply:
I got version 2.3.0 (latest stable) installed. I primarily use Arch Linux so you can consider everything I do as "tested with latest stable versions of every library" ^^